### PR TITLE
[D1] teach wrangler about Australia

### LINKED ISSUE #5505: 🐛 BUG: `d1 create` doesn't know about Australia
### Which Cloudflare product(s) does this pertain to?

D1

### What version(s) of the tool(s) are you using?

.

### What version of Node are you using?

.

### What operating system and version are you using?

.

### Describe the Bug

```
      --location  A hint for the primary location of the new DB. Options:
                  weur: Western Europe
                  eeur: Eastern Europe
                  apac: Asia Pacific
                  wnam: Western North America
                  enam: Eastern North America
```

### Please provide a link to a minimal reproduction

_No response_

### Please provide any relevant error logs

_No response_

### PR DESCRIPTION
**Pull request by rozenmd**
## What this PR solves / how to test
This PR teaches wrangler about Australia, and fixes up some loose ends.

Fixes #5505 

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [x] Cloudflare docs PR(s): https://github.com/cloudflare/cloudflare-docs/pull/13869
  - [ ] Not necessary because:

### CODE REVIEW

**Review by geelen — APPROVED** [review]

File: packages/wrangler/src/d1/create.tsx [hunk 1]
```diff
@@ -53,7 +53,6 @@ export const Handler = withConfig<HandlerOptions>(
 				},
 				body: JSON.stringify({
 					name,
-					experimental: true,
```
Comment (rozenmd) [inline packages/wrangler/src/d1/create.tsx:56]: the API no longer requires this - all created DBs have been using the new backend for about 8 months now

File: packages/wrangler/src/d1/create.tsx [hunk 2]
```diff
@@ -75,11 +74,7 @@ export const Handler = withConfig<HandlerOptions>(
 							? ` using primary location hint ${location}`
 							: ``}
 					</Text>
-					<Text>
-						Created your database using D1&apos;s new storage backend. The new
-						storage backend is not yet recommended for production workloads, but
-						backs up your data via point-in-time restore.
-					</Text>
+					<Text>Created your new D1 database.</Text>
```
Comment (rozenmd) [inline packages/wrangler/src/d1/create.tsx:77]: No more caveats.

File: packages/wrangler/src/d1/info.tsx [hunk 3]
```diff
@@ -45,6 +45,9 @@ export const Handler = withConfig<HandlerOptions>(
 			output["database_size"] = output["file_size"];
 			delete output["file_size"];
 		}
+		if (output["version"] !== "alpha") {
+			delete output["version"];
```
Comment (rozenmd) [inline packages/wrangler/src/d1/info.tsx:49]: The DB version is only relevant if it's alpha

**Review by petebacondarwin — APPROVED** [review]
Tests! ❤️

**Pull request merged**